website/modules/jumpToFragment.js

```diff
@@ -0,0 +1,20 @@
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
+
+export default (() => {
+  const isReady =
+    ExecutionEnvironment.canUseDOM && ExecutionEnvironment.canUseEventListeners;
+  if (isReady) {
+    window.addEventListener('load', () => {
+      if (!window.location.hash) {
+        return;
+      }
+
+      const ref = document.getElementById(window.location.hash.slice(1));
```

Comment:
As long as we're implementing this functionality in JS (boo! that we need to do that), what would it take to also support [Text Fragments](https://web.dev/text-fragments/)? There's an official [polyfill](https://github.com/GoogleChromeLabs/text-fragments-polyfill) from Chrome we might be able to reuse.

Reply:
Also, is there a way we can retrigger the browser's native implementation (by re-navigating to the current location?) instead of emulating it in JS?

Reply:
This is a really nice idea.  I'll add this as a followup idea worth investigating.